src/sentry/static/sentry/app/components/events/userFeedback.jsx

```diff
@@ -38,7 +42,12 @@ class EventUserFeedback extends React.Component {
                   <TimeSince date={report.dateCreated} />
                   <div className="activity-author">
                     {report.name}
-                    <small>{report.email}</small>
```

Comment:
Is the css that margined this element before safe to remove now?

Reply:
It's not, lots of other components still use this css